python/setup.py

```diff
@@ -92,22 +92,30 @@ def run(self):
             pass
         _sdist.run(self)
 
-
 class VWBuildExt(_build_ext):
     """Build pylibvw.so and install it as a python extension """
     def build_extension(self, ext):
         prep()
-        env = environ
-        env['PYTHON_VERSION'] = '{v[0]}.{v[1]}'.format(v=sys.version_info)
-        env['PYTHON_LIBS'] = '-l {}'.format(find_boost())
-        subprocess.check_call(['make', 'python'], cwd=path.join(here, 'src'), env=env)
         target_dir = path.dirname(self.get_ext_fullpath(ext.name))
         if not path.isdir(target_dir):
             makedirs(target_dir)
-        ext_suffix = 'so' if not system == 'Cygwin' else 'dll'
-        copy(path.join(here, 'src', 'python', '{name}.{suffix}'.format(name=ext.name, suffix=ext_suffix)),
-             self.get_ext_fullpath(ext.name))
-
+        if system == 'Windows':
```

Comment:
not sure if this is necessary here, the pyvw.dll copy should be handled by line 111, and the zlib.dll shouldn't be necessary if this is being pre-built and the lib is installed via InstallLib
Can you just extract lines 106-109 in the system != 'Windows' case?
Will need to extend the check for Cygwin or Windows at 110